backstop.json

```diff
@@ -0,0 +1,139 @@
+{
+  "id": "auro-input",
+  "viewports": [
+    {
+      "label": "tablet",
+      "width": 1024,
+      "height": 768
+    }
+  ],
+  "onBeforeScript": "playwright/onBefore.js",
+  "onReadyScript": "playwright/onReady.js",
+  "scenarios": [
+    {
+      "label": "default",
+      "url": "http://localhost:8000/demo/",
+      "referenceUrl": "",
+      "readyEvent": "",
+      "readySelector": "",
+      "delay": 0,
+      "hideSelectors": [],
+      "removeSelectors": [],
+      "hoverSelector": "",
+      "clickSelector": "",
+      "postInteractionWait": 0,
```

Comment:
Are these the default values for a lot of these? Would be good to try to slim some of these scenarios down if possible.